src/internal/model/model.go

```diff
@@ -27,7 +26,7 @@ const (
 
 // Valid returns true if the ModelType value fits within the iota range.
 func (mt Schema) Valid() bool {
-	return mt > 0 && mt < BackupDetailsSchema+1
+	return mt > 0 && mt < BackupSchema+1
```

Comment:
nit: maybe we should just make a sentinel value so we don't have to keep changing the code in two places when we update the set of model schemas we have